### DISCUSSION TITLE
Custom sizing, spacing and positioning of play area.

### DISCUSSION THREAD

**Discussion by Hissymaster** [ppy/osu#27872] [Ideas] [answered]
Maybe add the ability to have custom sizing or positioning of the play area. (for example, the size and orientation of the keys in mania, or the positioning of the play area in regular osu.)

**Comment by bdach** [comment, marked as answer]
already possible to a degree (gameplay scaling mode in settings, upscroll / downscroll in mania settings), will probably be expanded as skin editor is worked on

duplicate of probably at least 5 other discussions
